app/uk/gov/hmrc/customs/rosmfrontend/services/subscription/EnrolmentService.scala

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmrc.customs.rosmfrontend.services.subscription
+
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.customs.rosmfrontend.connector.{EnrolmentStoreProxyConnector, TaxEnrolmentsConnector}
+import uk.gov.hmrc.customs.rosmfrontend.domain.LoggedInUserWithEnrolments
+import uk.gov.hmrc.customs.rosmfrontend.models.enrolmentQuery.{EnrolmentAllocationRequest, EnrolmentRequest, KnownFactsQuery}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class EnrolmentService @Inject()(
+  enrolmentStoreProxyConnector: EnrolmentStoreProxyConnector,
+  taxEnrolmentsConnector: TaxEnrolmentsConnector
+)(implicit ec: ExecutionContext) {
+
+  // This will be retrieved from config, probably list of all possible services and retrieving the enrolment key
+  // Temporary solution
```

Comment:
Need to discuss this.   So far I've assumed the services we support (ATAR, GVMS, etc) will need to be hard-coded somewhere.   If we can make it config-driven that will be better.

Reply:
yeah, it will be hard coded, but I thought that hardcoding it in the config can be easier, but we can create some object or class with all those stuff inside